conf/jasminegraph-server.properties

```diff
@@ -31,7 +31,7 @@ org.jasminegraph.worker.path=/var/tmp/jasminegraph/
 #if it is empty please leave a space, else it will cause an issue
 org.jasminegraph.artifact.path=
 #org.jasminegraph.partitioner.metis.bin is the location where the METIS graph partitioner's gpmetis executable is installed
-org.jasminegraph.partitioner.metis.bin=/home/ubuntu/software/metis-5.1.0/bin
+org.jasminegraph.partitioner.metis.bin=/home/ubuntu/software/metis/metis-5.1.0/bin
```

Comment:
I think we need not have this line committed. This seems to be a private folder path right?